installing/cluster-capabilities.adoc

```diff
@@ -0,0 +1,74 @@
+:_content-type: ASSEMBLY
+[id="cluster-capabilities"]
+= Cluster capabilities
+include::_attributes/common-attributes.adoc[]
+:context: cluster-capabilities
+
+toc::[]
+
+Cluster administrators can use cluster capabilities to enable or disable optional components prior to installation. Cluster administrators can enable cluster capabilities at anytime after installation.
+
+[NOTE]
+====
+Cluster administrators cannot disable a cluster capability after it is enabled.
+====
+
+include::modules/selecting-cluster-capabilities.adoc[leveloffset=+1]
+
+include::snippets/capabilities-table.adoc[]
+
+[role="_additional-resources"]
+.Additional resources
+* xref:../installing/installing_aws/installing-aws-customizations.adoc#installing-aws-customizations[Installing a cluster on AWS with customizations]
+* xref:../installing/installing_gcp/installing-gcp-customizations.adoc#installing-gcp-customizations[Installing a cluster on GCP with customizations]
+
+include::modules/explanation-of-capabilities.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
+.Additional resources
+* xref:../operators/operator-reference.adoc#cluster-operator-reference[Cluster Operators reference]
+
+include::modules/cluster-bare-metal-operator.adoc[leveloffset=+2]
+
+[role="_additional-resources"]
+.Additional resources
+* xref:../installing/installing_bare_metal_ipi/ipi-install-overview.adoc#ipi-install-overview[Deploying installer-provisioned clusters on bare metal]
+* xref:../installing/installing_bare_metal/preparing-to-install-on-bare-metal.adoc#preparing-to-install-on-bare-metal[Preparing for bare metal cluster installation]
+* xref:../post_installation_configuration/bare-metal-configuration.adoc#post-install-bare-metal-configuration[Bare metal configuration]
+
+include::modules/cluster-storage-operator.adoc[leveloffset=+2]
+
+include::modules/console-operator.adoc[leveloffset=+2]
+
+[role="_additional-resources"]
+.Additional resources
+* xref:../web_console/web-console-overview.adoc#web-console-overview[Web console overview]
+
+include::modules/cluster-csi-snapshot-controller-operator.adoc[leveloffset=+2]
```

Comment:
Looks like all capabilities ordered by alphabetical, but is it possible to put **cluster-csi-snapshot-controller-operator** under **cluster-storage-operator**, as there are all storage related capability, especially choose to enable/disable them in this section.

Reply:
That's okay.